Doppler.MercadoPagoApi/Controllers/PaymentController.cs

```diff
@@ -1,15 +1,29 @@
+using Doppler.MercadoPagoApi.DopplerSecurity;
+using Doppler.MercadoPagoApi.Models;
+using Doppler.MercadoPagoApi.Services;
 using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
+using System.Threading.Tasks;
 
 namespace Doppler.MercadoPagoApi.Controllers
 {
     [Authorize]
     [ApiController]
     public class PaymentController : Controller
     {
-        public PaymentController()
+        private readonly IMercadoPagoService _mercadoPagoService;
+
+        public PaymentController(IMercadoPagoService mercadoPagoService)
         {
+            _mercadoPagoService = mercadoPagoService;
+        }
 
+        [Authorize(Policies.OWN_RESOURCE_OR_SUPERUSER)]
+        [HttpPost("/customers/{customername}")]
+        public async Task<IActionResult> CreateCustomerAsync(CreateCustomerRequest request)
+        {
+            var customer = await _mercadoPagoService.CreateCustomerAsync(request.Customer);
+            return Ok(new { CustomerId = customer.Id});
```

Comment:
Do you need this ID? maybe we can use the customername as id

Reply:
Yes, this id is used to store in the database. Currently we have the ```PaymentCustomerId``` column in the ```User``` table.

Reply:
By the moment we will try to understand the flow better ([see chat](https://makingsense.slack.com/archives/C03B7M42WHH/p1651773700904939))